hudi-integ-test/pom.xml

```diff
@@ -407,7 +407,46 @@
   </properties>
 
   <build>
+    <pluginManagement>
```

Comment:
Can you explain the need for this plugin ?

Reply:
scala classes were not part of build output if not for these.